mmdet/apis/ote/apis/detection/openvino_task.py

```diff
@@ -182,3 +209,61 @@ def evaluate(self,
                  output_result_set: ResultSet,
                  evaluation_metric: Optional[str] = None):
         return MetricsHelper.compute_f_measure(output_result_set).get_performance()
+
+    def optimize(self,
+                 optimization_type: OptimizationType,
+                 dataset: Optional[Dataset],
+                 output_model: OptimizedModel,
+                 optimization_parameters: Optional[OptimizationParameters]):
+
+        if optimization_type is not OptimizationType.POT:
+            raise RuntimeError("POT is the only supported optimization type for OpenVino models")
+
+        data_loader = ScOpenVinoDataLoader(dataset, self.inferencer)
+
+        with tempfile.TemporaryDirectory() as tempdir:
+            xml_path = os.path.join(tempdir, "openvino.xml")
+            bin_path = os.path.join(tempdir, "openvino.bin")
+            open(xml_path, "wb").write(self.model.get_data("openvino.xml"))
+            open(bin_path, "wb").write(self.model.get_data("openvino.bin"))
+
+            model_config = ADDict({
+                'model_name': 'detection',
```

Comment:
Could it be a configurable model name or not? The same for .xml's and .bin's file names.